src/pythie_serving/table_wrapper.py

```diff
@@ -19,6 +20,18 @@
 class TablePredictionServiceServicer(AbstractPythieServingPredictionServiceServicer):
     model_file_extension = ".csv"
 
+    def _parse_extra_specs(self, metadata: dict[str, Any], table_type_mapping: dict[str, Any]):
```

Comment:
Missing a return type, but should have been catched by the CI ...
Could you still add it ?

Reply:
I think this is also a mypy configuration issue. I have no warning when running mypy in local